R_D_HW_9/homework_9_generator.py

```diff
@@ -0,0 +1,12 @@
+def fibonacci_generator():
+    a, b = 0, 1
+    while True:
+        yield a
+        a, b = b, a + b
+
+while True:
+    fib = fibonacci_generator()
+    n = int(input("Enter a number: "))
+    for i in range(n):
+        next_fib = next(fib)
+    print(next_fib)
```

Comment:
Your program print a wrong number in the sequence. For example, for 7 it should print 13 but it prints 8.